test/JDBC/expected/babel_varbinary.out

```diff
@@ -164,6 +164,323 @@ go
 ~~ERROR (Message: division by zero)~~
 
 
+--division between varbinary and int4 datatype, vice-versa
```

Comment:
some test cases are still missing from what we suggested: varbinary --> length > 4 bytes, length = 0, binary > INT4_MAX, binary < INT4_MIN, binary = NULL . same for int as well.

Reply:
For int we need to be in range of INT_MIN and INT_MAX for this particular operator to hit .
For varbinary length greater than 4 bytes is giving wrong value while casting it to int4.
For binary , we are have some cast issue , created  jira for that , will add test case which can be updated in future.